rdrf/rdrf/models/definition/models.py

```diff
@@ -1499,6 +1499,7 @@ class ContextFormGroup(models.Model):
                                  related_name="context_form_groups",
                                  on_delete=models.CASCADE)
     context_type = models.CharField(max_length=1, default="F", choices=CONTEXT_TYPES)
+    code = models.CharField(max_length=30, unique=True)
```

Comment:
The other models with `code` / `name` field check that they're alphanumeric in their `clean` methods.

Reply:
Fixed but I need to regenerate fh.zip to apply the modified migration of the default CFG Code values.

Reply:
done.